### PR TITLE
oldification things

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Oldification for guns and tools are now always give them a mod that reduces stats
Adds more different bad oldification mods to the game

Turns old edits of oldificaiton has never worked if you ever wielded the tool to reset it

### CODE REVIEW

**Comment by MikeTheWallDweller** [comment]
Couple grammatical errors here and there, might be worth going over before merging

**Comment by Smokingsquirrel** [comment]
> Couple grammatical errors here and there, might be worth going over before merging

yeah yeah i am on it

File: code/modules/projectiles/guns/mods/mods_for_old.dm [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+	)
+	I.destroy_on_removal = TRUE
+	I.removal_time *= rand(10, 14)/10
+	I.removal_difficulty *= rand(5, 15)/10
+	I.gun_loc_tag = GUN_BARREL
+	I.prefix = "fluted barrel" // In line with forged barrel
+
+//More power + less pen
+/obj/item/gun_upgrade/muzzle/rusted
+	name = "Rusted Copper Muzzel"
```
Comment (Smokingsquirrel) [inline code/modules/projectiles/guns/mods/mods_for_old.dm:168]:
```suggestion
	name = "Rusted Copper Muzzle"
```

File: code/modules/projectiles/guns/mods/mods_for_old.dm [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+	desc = "A copper muzzel that has been water damaged. This causes the attached weapon to suffer from weaker armor penetration and increases power draw."
```
Comment (Smokingsquirrel) [inline code/modules/projectiles/guns/mods/mods_for_old.dm:169]:
```suggestion
	desc = "This copper muzzle is rusted from water exposure. This causes the weapon it is attached to to suffer weaker armor penetration and increased power drain."
```

File: code/modules/projectiles/guns/mods/mods_for_old.dm [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+	I.destroy_on_removal = TRUE
+	I.removal_time *= rand(10, 14)/10
+	I.removal_difficulty *= rand(5, 15)/10
+	I.gun_loc_tag = GUN_MUZZLE
+	I.prefix = "heatsunk"
+
+//Massively increases charge costs
+/obj/item/gun_upgrade/mechanism/outdated
+	name = "Outdated Charge Port Mechanism"
+	desc = "An outdated old copper based charge port that makes any almost hold half as much charge."
```
Comment (Smokingsquirrel) [inline code/modules/projectiles/guns/mods/mods_for_old.dm:190]:
```suggestion
	desc = "An outdated copper based charge port that massively increases power drain."
```

File: code/modules/projectiles/guns/mods/mods_for_old.dm [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+	)
+	I.destroy_on_removal = TRUE
+	I.removal_time *= rand(10, 14)/10
+	I.removal_difficulty *= rand(5, 15)/10
+	I.gun_loc_tag = GUN_MECHANISM
+	I.prefix = "ticking"
+
+//Egun verson of the rigged loader
+/obj/item/gun_upgrade/mechanism/bad_brackets
+	name = "Acid Corroided Brackets"
```
Comment (Smokingsquirrel) [inline code/modules/projectiles/guns/mods/mods_for_old.dm:209]:
```suggestion
	name = "Corroded Brackets"
```

File: code/modules/projectiles/guns/mods/mods_for_old.dm [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+	desc = "Everything that makes sure the laser goes the right way was damaged by cell acid making it likely to discharge backwards at a luckly reduced rate."
```
Comment (Smokingsquirrel) [inline code/modules/projectiles/guns/mods/mods_for_old.dm:210]:
```suggestion
	desc = "Cell Acid leaked and corroded the coils for stabilizing energy projectiles. Has a tendency to discharge the gun in the wrong direction and damage the user."
```

File: code/modules/projectiles/guns/mods/mods_for_old.dm [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+		UPGRADE_COLOR = pick("#AA7744", "#774411", "#777777")
+	)
+	I.destroy_on_removal = TRUE
+	I.removal_time *= rand(10, 14)/10
+	I.removal_difficulty *= rand(5, 15)/10
+	I.gun_loc_tag = GUN_BARREL
+	I.prefix = "fluted barrel" // In line with forged barrel
+
+/obj/item/gun_upgrade/muzzle/faulty
+	name = "Failed Makeshift Silencer"
```
[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply code/modules/projectiles/guns/mods/mods_for_old.dm:45]: This one was already like this, and I like it being a failed product

File: code/game/objects/items/weapons/tools/mods/trash_mods.dm [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+	UPGRADE_COLOR = pick("#AA7744", "#774411", "#777777")
+	)
+	I.prefix = "heatsunk"
+	I.destroy_on_removal = TRUE
+	I.removal_time *= rand(10, 14)/10
+	I.removal_difficulty *= rand(5, 15)/10
+
+/obj/item/tool_upgrade/not_magnetic
+	name = "demagnetized bit"
+	desc = "Someone welded this on with too much heat and destroyed the the bit in the process. This is useless."
```
Comment (Smokingsquirrel) [inline code/game/objects/items/weapons/tools/mods/trash_mods.dm:130]:
```suggestion
	desc = "Someone welded this on with too much heat and destroyed the bit in the process. This is useless."
```

File: code/modules/projectiles/guns/mods/mods_for_old.dm (hunk 6 above)
[the remark this answers is not in the document]
Comment (Smokingsquirrel) [reply code/modules/projectiles/guns/mods/mods_for_old.dm:45]: It’s still failed

**Pull request merged**